_observing-your-data/ad/index.md

```diff
@@ -110,7 +110,7 @@ To get the entity model size of a detector, use the [profile detector API]({{sit
 This formula provides a good starting point, but make sure to test with a representative workload.
```

Comment:
Can you also add here:

For a detailed and comprehensive exploration of this subject, please visit our guide on [How to Size Domains](https://aws.amazon.com/blogs/big-data/detect-anomalies-on-one-million-unique-entities-with-amazon-opensearch-service/).

@hdhalter please take a look as well.

Reply:
Thanks, @kaituo! But since we are open source, we cannot reference AWS content. Maybe there's an equivalent blog on the opensearch.org website?

Reply:
There is somewhat related blog in open source https://opensearch.org/blog/one-million-enitities-in-one-minute/

Reply:
> There is somewhat related blog in open source https://opensearch.org/blog/one-million-enitities-in-one-minute/

Hi @kaituo, I did add the content you mentioned.